Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -1794,6 +1803,11 @@ open class NavigationMapView: UIView {
                    layerInfo.type.rawValue != "symbol",
                    let sourceLayer = mapView.mapboxMap.style.layerProperty(for: layerInfo.id, property: "source-layer").value as? String,
                    !sourceLayer.isEmpty {
+                    if layerInfo.type.rawValue == "circle",
```

Comment:
According to #4079 , add check for `circle` type label layer, route line will be placed below them.